modules/oe_translation_poetry/modules/oe_translation_poetry_html_formatter/tests/Kernel/HtmlFormatterTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation_poetry_html_formatter\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+
+/**
+ * Tests the Poetry Html Formatter.
+ */
+class HtmlFormatterTest extends KernelTestBase {
+
+  use UserCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'path',
+    'field',
+    'options',
+    'text',
+    'node',
+    'user',
+    'system',
+    'filter',
+    'language',
+    'oe_multilingual',
+    'content_translation',
+    'views',
+    'tmgmt',
+    'tmgmt_content',
+    'tmgmt_test',
+    'oe_translation_poetry_html_formatter',
+  ];
+
+  /**
+   * The job to test.
+   *
+   * @var \Drupal\tmgmt\JobInterface
+   */
+  protected $job;
+
+  /**
+   * The job item to test.
+   *
+   * @var \Drupal\tmgmt\JobItemInterface
+   */
+  protected $jobItem;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    $this->installConfig([
+      'system',
+      'node',
+      'field',
+      'user',
+      'language',
+      'oe_multilingual',
+    ]);
+
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('tmgmt_job');
+    $this->installEntitySchema('tmgmt_job_item');
+
+    $this->installSchema('node', 'node_access');
+    $this->installSchema('system', 'sequences');
+
+    $node_type = $this->container->get('entity_type.manager')->getStorage('node_type')->create([
+      'name' => 'Test node type',
+      'type' => 'test_node_type',
+    ]);
+
+    $node_type->save();
+
+    tmgmt_translator_auto_create(\Drupal::service('plugin.manager.tmgmt.translator')->getDefinition('test_translator'));
```

Comment:
!`\Drupal::service` but `$this->container..`. Also, please add a comment above to say what this does.